libraries/standard/http/utest/CMakeLists.txt

```diff
@@ -25,13 +25,15 @@ list(APPEND mock_define_list
 # list the files you would like to test here
 list(APPEND real_source_files
             ${HTTP_SOURCES}
+            "${MODULES_DIR}/standard/http/third_party/http_parser/http_parser.c"
```

Comment:
We would need these to be mocked instead of building the source of the 3rd party. 
(However, that could be handled in a separate PR)